### DISCUSSION TITLE
can i make maddy send emails that fail security checks(SPF, DKIM, DMARC...) to the Junk inbox instead of rejecting them?

### DISCUSSION THREAD

**Discussion by Damaj301damaj-lol** [emersion/maddy#679] [Q&A] [answered]
I have to use some services which fail these security checks, and i am not willing to disable these checks for these services, can i make these emails go to the Junk folder of the recipient instead of the void(hehe, aka rejected or quarantined)

Thanks

**Comment by foxcpp** [comment, marked as answer]
"quarantined" means exactly what you want - going into Junk folder.
  
If DMARC policy says to reject or quarantine failing messages, this is what will be done - currently, there is no option to override this.

If there is no DMARC policy for the sender domain, action taken depending on SPF and DKIM results can be configured. By default, broken or missing DKIM signature does not cause the message to be rejected or quarantined. For SPF, action taken on softfail/fail/etc is configured using corresponding check.spf directives: https://maddy.email/reference/checks/spf/

**Reply by Damaj301damaj-lol** [reply]
I never refreshed my junk folder thinking it was useless, thanks dude 😅

**Reply by Damaj301damaj-lol** [reply]
> "quarantined" means exactly what you want - going into Junk folder.
>   
> If DMARC policy says to reject or quarantine failing messages, this is what will be done - currently, there is no option to override this.
> 
> If there is no DMARC policy for the sender domain, action taken depending on SPF and DKIM results can be configured. By default, broken or missing DKIM signature does not cause the message to be rejected or quarantined. For SPF, action taken on softfail/fail/etc is configured using corresponding check.spf directives: https://maddy.email/reference/checks/spf/ 

@foxcpp how can i add the specific directives to the maddy.conf file?
I cannot find under which parent to add the check.dkim directive.
